tests/integration.rs

```diff
@@ -173,23 +191,63 @@ fn stamp_file() -> TestResult {
 
     // For some reason, we delete the stamp file after `--file` :(
     // Recreate it.
-    run(sweep(&["--stamp"]));
+    run(sweep(&["--stamp"]), target.path());
 
     let actual_cleaned = count_cleaned(&target, args, size)?;
     assert_eq!(actual_cleaned, expected_cleaned);
 
     Ok(())
 }
 
+#[test]
+fn empty_project_output() -> TestResult {
+    let (_size, target) = build("sample-project")?;
+
+    let assert = run(
+        sweep(&["--maxsize", "0"]).current_dir(test_dir().join("sample-project")),
+        target.path(),
+    );
+
+    let output = std::str::from_utf8(&assert.get_output().stdout).unwrap();
+
+    let regex_skip = r#".+?"#;
```

Comment:
this can just be `.*` instead of needing to make it optional; and at that point I think it would be simpler to inline it instead of using `format`.

Reply:
Aha! A regex _Gotcha_.

Fun fact: Inserting an `?` after a quantifier (`*`, `+`, `?` or `{N..M}`) switches
to non-greedy matching, which can be good for performance as it decreases
excessive backtracking from the usual greedy eater.

(It's the same symbol to make stuff optional... why...)

Well, that's a tiny text to match so I'll switch to `.+` anyways :+1:.

Reply:
I do think this is a bit less readable from the editor, but the GitHub diff doesn't highlight the format placeholder.

![image](https://user-images.githubusercontent.com/38900226/209582630-059b71c3-4809-4bff-adec-b6dd4ac88d89.png)

![image](https://user-images.githubusercontent.com/38900226/209582685-16d4a565-5b21-4f5c-9ac9-0d3d8d23e017.png)